frontend/src/app/leaderboard/topStats.tsx

```
@@ -0,0 +1,64 @@
+const topStats = () => {
+  return (
+    <div className="flex mt-5 justify-center">
+      <div className="stats border-4 border-[#FF5B36]">
+        <div className="stat">
+          <div className="stat-figure hidden md:block text-primary">
+          <svg
+              xmlns="http://www.w3.org/2000/svg"
+              fill="none"
+              viewBox="0 0 24 24"
+              className="inline-block h-8 w-8 stroke-current"
+            >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)